### PR TITLE
Add suggestion how to convert the distro in step 12

### PR DESCRIPTION
**Pull request by velletti**
needed ,if it was created earlier as WLS 1 distro.

just added : 

If not , and your linux distro was created earlier with version 1, you can convert it in Powershell: `wsl --set-version <distro> 2`. Replace `<distro>` with the shown distro

## The Problem/Issue/Bug:
Running in Error in step 12 as my distro was set up more thatn 12 month ago without wsl --set-default-version 2 before

## How this PR Solves The Problem:
just tried it once and helped me

## Automated Testing Overview:
as it is an optional step, only needed when Step 12 does not show the  wanted : all distros have version 2 

## Release/Deployment notes:

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3107"></a>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -74,7 +74,8 @@ We'll walk through these in more detail. You may prefer other techniques of inst
...
 11. Go to Docker Desktop settings > Resources > WSL integration > enable integration for your distro (now `docker` commands will be available from within your WSL2 distro).
-12. Double-check in PowerShell: `wsl -l -v` should show three distros, and your Ubuntu should be the default. All three should be WSL version 2.
+12. Double-check in PowerShell: `wsl -l -v` should show three distros, and your Ubuntu should be the default. All three should be WSL version 2. 
+If not , and your linux distro was created earlier with version 1, you can convert it in Powershell: `wsl --set-version <distro> 2`. Replace `<distro>` with the shown distro name, that still uses version 1.
```
Comment (rfay) [inline docs/index.md:78]: I'm not entirely sure this is our job to coach this here, but I'll think about it. Unfortunately adding solutions for every single possible situation can really clutter up docs.

Comment (rfay) [reply docs/index.md:78]: Really appreciate your contribution and hope you'll fix other things you find in the docs!

**Comment by tyler36** [comment]
Step 9 references distro with [Ubuntu] recommended. Step 12 references Ubuntu but also  ....
``wsl --set-version <distro> 2`. Replace `<distro>``

For consistency,  I suggest the following changes 
```diff
- 9. Install a WSL2 distro from the Microsoft Store. We recommend [Ubuntu]
+ 9. Install a WSL2 `<distro>` from the Microsoft Store. We recommend [Ubuntu]
...
- 12. Double-check in PowerShell: `wsl -l -v` should show three distros, and your Ubuntu should be the default. All three should be WSL version 2. 
+ 12. Double-check in PowerShell: `wsl -l -v` should show three distros, and your `<distro>` should be the default. All three should be WSL version 2.

**Comment by rfay** [comment]
I very much appreciate you taking the time to add this, but after consideration I don't think it belongs in ddev docs. 

Also, I recommend to people to create a new distro rather than converting one. It takes absolutely forever to convert one, and who knows what might go wrong.

**Pull request closed without merging**